### PR TITLE
feat: Add sample notebook to build an AI podcasting agent using LangGraph and Gemini

### PR DESCRIPTION
**Pull request by koverholt**
# Description

This PR adds a sample notebook that uses LangGraph and Gemini to build an AI agent to research, write, and refine a podcast script and then generate audio via TTS APIs.

- [X] Follow the [`CONTRIBUTING` Guide](https://github.com/GoogleCloudPlatform/generative-ai/blob/main/CONTRIBUTING.md).
- [X] You are listed as the author in your notebook or README file.
  - [X] Your account is listed in [`CODEOWNERS`](https://github.com/GoogleCloudPlatform/generative-ai/blob/main/.github/CODEOWNERS) for the file(s).
- [X] Make your Pull Request title in the <https://www.conventionalcommits.org/> specification.
- [X] Ensure the tests and linter pass (Run `nox -s format` from the repository root to format).
- [X] Appropriate docs were updated (if necessary)

### CODE REVIEW

**Comment by koverholt** [comment]
Note that the spelling check / forbidden pattern that is failing is because I'm referring to `arxiv` the library and in a node function name without the usual stylization of arXiv. Any way to flag to the spell checker that this is an expected/known failure in this notebook?

**Comment by koverholt** [comment]
Oh, nevermind, it was catching the actual typed Arxiv in the narrative text, thanks for catching/fixing!

File: gemini/orchestration/langgraph_gemini_podcast.ipynb [hunk 1]
```diff
@@ -0,0 +1,1430 @@
...
+    {
+      "cell_type": "code",
+      "execution_count": 9,
+      "metadata": {
+        "id": "0d27ed8a91c1"
+      },
+      "outputs": [],
+      "source": [
+        "@tool\n",
+        "def search_arxiv(query: str):\n",
```
Comment (holtskinner) [inline gemini/orchestration/langgraph_gemini_podcast.ipynb:413]: nit: Can you add typing for the return values of these methods? Can be easier to read/understand.

**Review by holtskinner — APPROVED** [review]

File: gemini/orchestration/langgraph_gemini_podcast.ipynb (hunk 1 above)
Comment (koverholt) [reply gemini/orchestration/langgraph_gemini_podcast.ipynb]: Done and added, thanks for the review!

**Pull request merged**